src/content/docs/features/chwd.md

```diff
@@ -0,0 +1,92 @@
+---
+title: CachyOS Hardware Detection
+description: Hardware Detection and Configuration for CachyOS
+---
+
+[CachyOS Hardware Detection](https://github.com/CachyOS/chwd/) or better known as **`chwd`** is our hardware detection tool that
+enables us to power a variety of hardware.
+This includes systems running NVIDIA's graphics cards, T2 Macbooks and handheld devices such as Steam Deck and ROG Ally.
+
+## Usage
+
+**`chwd`** is typically ran during installation time to provide the necessary packages for your system. However, it is also possible
+to use it post-install.
+
+### Auto Configuration via Class IDs
+
+We can get class IDs by running `lspci`
+
+```sh
+❯ lscpi -nn
+01:00.0 VGA compatible controller [0300]: NVIDIA Corporation GA107BM [GeForce RTX 3050 Mobile] [10de:25e2] (rev a1)
+#                                  ^ ^ ^ -- This is the class ID!
+```
+
+After finding the necessary class ID, we can pass it to chwd to install **all** the necessary profiles for your system.
+
+```sh
+❯ sudo chwd -a pci nonfree 0300
```

Comment:
This command doesn't quite match the current behavior in master, and should probably be changed with the next chwd release.

Reply:
sure

Reply:
Yes, but we still didnt make release. Its fine for current stable, will edit wiki after